src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,22 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import mate.academy.spring.validation.ValidPassword;
+
+@ValidPassword
 public class UserRequestDto {
+
+    @Email
+    @NotEmpty
```

Comment:
<img width="988" alt="image" src="https://github.com/mate-academy/jv-spring-security/assets/81781479/b647f0ca-317b-4789-a0c5-b177e1cfc66f">

Reply:
I thought this annotations are different...